src/grpc_server.cc

```diff
@@ -139,15 +142,16 @@ namespace grpc_labview
         bool ok;
         while (true)
         {
+            if (_shutdown)
```

Comment:
Move the flag check before checking the completion because it is possible that users tries to close the server very quickly so when we enter this while loop, the server and completion queue are already shutdown.